src/ts/contentScripts/shortcuts/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { browser } from 'webextension-polyfill-ts';
+import { getSetting, isActive, Feature, Shortcut } from '../../utils/settings';
+import { features } from '../features'
+
+
+export const isShortcutPressed = (shortcut: string, event: KeyboardEvent) => {
```

Comment:
I'm pretty sure that there are JS libs out there to handle keyboard shortcut stuff. I'd rather we use one of them :)

Reply:
There are js libs, none works for me, you can try yourself, either I'm doing something wrong or you need to have everythings that works with document stuff in content scripts. I couldn't even put that simple code in any other place than another CS.

Reply:
If it doesn't work for you too maybe it will work when injected into Roam page as <script> from assets, didn't try that

Reply:
>you need to have everything that works with document stuff in content scripts
I believe it's the case. You can still use libraries from content scripts though (see fuzzydate stuff as an example)

Reply:
You're using fuzzydate when you're on roam website, but one can change shortcuts right after installing ext or on another website, I might try that on FF debugging page, maybe that was the reason it didn't work.

Reply:
Nah, I tried that on roam website too, for sure. Just try it if you want and let me know, it was one the most annoying things, I tried hotkeys, mousetrap and one more, don't remember.

Reply:
will try to build prototype tomorrow and let you know how it went!

Reply:
Tested it again ([hotkeys-js](https://github.com/jaywcjlove/hotkeys)), in srs/index.tsx so nothing should interfere there, it works now (i hate it so much 💔), but only when I'm not editing block. Maybe you will be able to apply some magic to that, I don't know else can be done.

Reply:
hotkey-js works. the input omission is explicitly done by it https://github.com/jaywcjlove/hotkeys#user-content-filter and can be disabled in configuration

Reply:
also while looking at it stumbled upon https://github.com/greena13/react-hotkeys which seems a bit complex to start with but provides react integration and things like recording shortcuts from users pressing them /etc